src/shared/markdown-it/stack-language-comments.ts

```diff
@@ -1,6 +1,5 @@
-import MarkdownIt from "markdown-it/lib";
-import StateBlock from "markdown-it/lib/rules_block/state_block";
-import State from "markdown-it/lib/rules_core/state_core";
+import MarkdownIt from "markdown-it";
+import { StateCore, StateBlock } from "markdown-it";
```

Comment:
We might want to inline this import statement for consistency with the other files.